proposals/dictionary-expressions.md

```diff
@@ -185,7 +246,7 @@ An implicit *collection expression conversion* exists from a collection expressi
 * A *dictionary type* with an appropriate *[create method](#create-methods)*.
 * A *struct* or *class* *dictionary type* that implements `System.Collections.IEnumerable` where:
   * The *element type* is determined from a `GetEnumerator` instance method or enumerable interface.
-  * The *type* has an *[applicable](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#11642-applicable-function-member)* constructor that can be invoked with no arguments, and the constructor is accessible at the location of the collection expression.
+  * The *type* has an *[applicable](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#11642-applicable-function-member)* constructor that can be invoked with no arguments (*or* a constructor with an `IEqualityComparer<TKey>` parameter), and the constructor is accessible at the location of the collection expression.
```

Comment:
What would we like the compiler to do when the `comparer` parameter type is a subtype of `IEqualityComparer<TKey>`, e.g. `new MyStringDictionary(StringComparer comparer)`?

Reply:
Good point.  this should be something assignable to `IEQ<TKey>`